src/components/menu-left/MenuLeft.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="menu-left">
+    <b-field :label="`Show: ${title}`" data-test="menu-left--title" />
```

Comment:
nit: We might as well just make this a header! We don't need `b-field` since there's no form input to wrap here! 

```tsx
<h2 data-test="menu-left--title">Show: ${title}</h2>
```

If you need, you can create a CSS class to replicate the label's style.

Reply:
wow, I have no idea how to do this... 😅

Reply:
How about we track as a future item with #149?